### PR TITLE
Resolve issue 30

### LINKED ISSUE #30: Landing page: headline should disclose status of project
The landing page should mention where the project stands in its development and deployment. As of now, it reads like this is a completed project.

### PR DESCRIPTION
**Pull request by alexandercyu**
Add message telling visitors the site is under construction and more content is coming.

Resolves #30 .

### CODE REVIEW

File: static/css/daccs-css.css [hunk 1]
```diff
@@ -177,6 +174,15 @@
   -webkit-text-fill-color: transparent;
 }
 
+/*Page Elements*/
+#nodeContentDisplay{
```
Comment (mishaschwartz) [inline static/css/daccs-css.css:178]: How are these related to this PR? I don't see them used anywhere here.

File: templates/partials/alert.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="col-12 position-sticky mt-5 construction-alert">
+        <p class="m-auto text-center fs-3">Check back for more updates!</p>
```
Comment (mishaschwartz) [inline templates/partials/alert.html:2]: #30 points out that there should be some mention about the project status. This doesn't really convey that information

What about:  "The Marble project is in active development. Check back here for updates."

File: templates/partials/banner.html [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <script src="//unpkg.com/globe.gl"></script>
 <div class="row mb-3 banner-text-container">
+    {% include "partials/alert.html" %}
```
Comment (mishaschwartz) [inline templates/partials/banner.html:3]: I don't think that the banner partial is the right place for this. Not all pages have a banner (ie. the tutorials page). 

I think that it should go after the menu partial declaration on the layouts/main.html file:

```
<body>
{% include "partials/menu.html" %}
{% include "partials/alert.html" %}
{% block content %} {% endblock %}
{% include "partials/footer.html" %}
</body>
```

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

File: static/css/daccs-css.css (hunk 1 above)
Comment (alexandercyu) [reply static/css/daccs-css.css:178]: They're not.  I just moved them from their previous location because I felt the order of the classes should more or less follow page elements.  So I moved them in between the body classes and footer.

File: templates/partials/alert.html (hunk 2 above)
Comment (alexandercyu) [reply templates/partials/alert.html:2]: That sums it up.

Changed the text to the message.

File: templates/partials/banner.html (hunk 3 above)
Comment (alexandercyu) [reply templates/partials/banner.html:3]: Because of how the HTML is done putting it here will push down the banner instead of floating on top of it.

Comment (alexandercyu) [reply templates/partials/banner.html]: Moved alert partial to main.html

**Review by mishaschwartz — APPROVED** [review]
